From 1b2cc62c0802af650f80eebcc716b5d5db87030e Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 8 Aug 2022 13:56:49 -0400
Subject: [PATCH] Issue 5397 - Fix check pick error

Description:

Original commit included a free for a new filter, but that filter was
not implemented in 1.4.3
---
 ldap/servers/slapd/back-ldbm/ldbm_search.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldap/servers/slapd/back-ldbm/ldbm_search.c b/ldap/servers/slapd/back-ldbm/ldbm_search.c
index 771c35a33..d0f52b6f7 100644
--- a/ldap/servers/slapd/back-ldbm/ldbm_search.c
+++ b/ldap/servers/slapd/back-ldbm/ldbm_search.c
@@ -1930,7 +1930,6 @@ delete_search_result_set(Slapi_PBlock *pb, back_search_result_set **sr)
                       rc, filt_errs);
     }
     slapi_filter_free((*sr)->sr_norm_filter, 1);
-    slapi_filter_free((*sr)->sr_norm_filter_intent, 1);
     memset(*sr, 0, sizeof(back_search_result_set));
     slapi_ch_free((void **)sr);
     return;
-- 
2.37.1

